src/tools/illink/src/ILLink.RoslynAnalyzer/TrimAnalysis/TrimAnalysisReturnValuePattern.cs.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using ILLink.Shared;
+using ILLink.Shared.DataFlow;
+using ILLink.Shared.TrimAnalysis;
+using ILLink.RoslynAnalyzer.DataFlow;
+using Microsoft.CodeAnalysis;
+
+namespace ILLink.RoslynAnalyzer.TrimAnalysis
+{
+	public readonly record struct TrimAnalysisReturnValuePattern
```

Comment:
What does this represent? Is it always the return value of a [FeatureCheck] property in a method, or [FeatureCheck] property getter definition?